.azure-pipelines/scripts/Get-SigntoolPath.ps1

```diff
@@ -0,0 +1,26 @@
+function Get-Signtool() {
+  <#
+    .SYNOPSIS
+    Function used to get signtool from windows SDK
+  #>
+
+  [System.IntPtr]::Size # an integer with platform-specific size.
+  $systemBit = "x64"
+  $programFiles = ${Env:ProgramFiles(x86)}
+
+  if([System.IntPtr]::Size -eq 4) {
+    $systemBit = "x86"
+    $programFiles = ${Env:ProgramFiles}
+  }
```

Comment:
Could it make sense to move the logic of determining the OS architecture into a separate function?
Also, maybe it's better to query information about architecture from WMI or CIM classes?
```PowerShell
(Get-WmiObject Win32_OperatingSystem).OSArchitecture
(Get-CimInStance CIM_OperatingSystem).OSArchitecture
(Get-WmiObject Win32_Processor).AddressWidth
```

Reply:
Good point changed the logic, now I am using of `(Get-WmiObject Win32_Processor).AddressWidth`  command